region/crateria/east.json

```diff
@@ -1167,6 +1167,43 @@
                 }
               ]
             },
+            {
+              "id": 12,
+              "strats": [
+                {
+                  "name": "G-Mode Deep Stuck X-Ray Climb",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [4],
```

Comment:
Does the little bit of knockback keep you from falling?

Reply:
Seems the knockback doesn't work. Added requirements to get back up. Rather than duplicating all 5 -> 4 strats, I just put two options in "canArtificialMorph" and "canWalljump" since if the player is enabling G-mode they're very likely enabling at least one of those two.